package.json

```diff
@@ -5,9 +5,9 @@
   },
   "type": "commonjs",
   "devDependencies": {
+    "@types/node": "^18.6.4",
```

Comment:
did you move this or a tool? Only I've seen tools move it the other way around before.

Reply:
I guess this was due to a merge conflict. So maybe I did that.